packages/redux_persist_flutter/lib/redux_persist_flutter.dart

```diff
@@ -128,7 +128,7 @@ class FlutterJsonSerializer<T> implements StateSerializer<T> {
   @override
   Future<T?> decode(Uint8List? data) async {
     return decoder(
-        data != null ? json.decode(await uint8ListToString(data) ?? '') : null);
+        data != null ? json.decode((await uint8ListToString(data))!);
```

Comment:
This is not quite right. You accidentally removed the null case in the ternary operator.

Reply:
Whoops